imap_processing/glows/l1b/glows_l1b.py

```diff
@@ -104,7 +211,7 @@ def process_histogram(l1a: xr.Dataset) -> xr.Dataset:
     dataarrays = [l1a[i] for i in l1a.keys()]
 
     dims = [[] for i in l1a.keys()]
-    # 34 is the number of output attributes in the HistogramL1B class.
+    # 35 is the number of output attributes in the HistogramL1B class.
```

Comment:
Do you need to update the range(34) call below too then? There is mismatch between 34/35 here.

Reply:
The code is correct, the number is wrong. I might just remove the number from the comment because it's confusing.